process/transaction/shardProcess.go

```diff
@@ -522,6 +501,7 @@ func (txProc *txProcessor) processRelayedTx(
 	if !bytes.Equal(userTx.SndAddr, tx.RcvAddr) {
 		return vmcommon.UserError, txProc.executingFailedTransaction(tx, relayerAcnt, process.ErrRelayedTxBeneficiaryDoesNotMatchReceiver)
 	}
+	//TODO: check why this check? Why can't the relayer send a bigger value?
```

Comment:
why would you want to send more value to user ? if you want to give value to user you can simply transfer to him.

Reply:
ok